File: b_rank_skillcheck_archive/plural_form_03.cs
// 実行時間 20ms
using static System.Console;

internal class Program
{
    static void Main()
    {
        int n = int.Parse( ReadLine() );

        for( int _ = 0; _ < n; _++ ) {
            string a = ReadLine();

            if( a.EndsWith( "f" ) ) {
                WriteLine( $"{a.Substring( 0, a.Length - 1 )}ves" );
            } else if( a.EndsWith( "fe" ) ) {
                WriteLine( $"{a.Substring( 0, a.Length - 2 )}ves" );
            } else {
                WriteLine( $"{a}s" );
            }
        }
        return;
    }
}

File: b_rank_skillcheck_archive/plural_form_03.py
# 実行時間 10ms
n = int(input())

for _ in [0] * n:
    a = input()

    if a.endswith("f"):
        print(f"{a[:-1]}ves")
    elif a.endswith("fe"):
        print(f"{a[:-2]}ves")
    else:
        print(f"{a}s")
